### PR TITLE
Fix/communication pmpy

### PR DESCRIPTION
**Pull request by lagoan**
## Context

pushmi_pullyu has not been communicating with jupiter because the entries on the Redis queue did not have the proper format. This PR simply adds the required information from jupiter to Redis.

Related to #2527

## What's New

The json values for Redis entered when a new item or thesis is saved now include the id and the type of entity that was saved.

### CODE REVIEW

**Review by murny — APPROVED** [review]
LGTM 👍 . Nice investigation into the problem

File: app/models/jupiter_core/depositable.rb [hunk 1]
```diff
@@ -159,19 +159,21 @@ def add_to_embargo_history
...
-  def push_item_id_for_preservation
+
+  def push_entity_for_preservation
     queue_name = Rails.application.secrets.preservation_queue_name
 
     $queue ||= ConnectionPool.new(size: 1, timeout: 5) { Redis.current }
 
     $queue.with do |connection|
-      connection.zadd queue_name, Time.now.to_f, id
+      entity = { uuid: id, type: self.class.table_name }
```
Comment (pgwillia) [inline app/models/jupiter_core/depositable.rb:169]: Should we add a comment that this needs to match [pushmi_pullyu](https://github.com/ualbertalib/pushmi_pullyu)?

**Review by pgwillia — APPROVED** [review]
Thanks again for your investigation of this.

**Review by pgwillia — APPROVED** [review]
👍

**Pull request merged**